Package/gulpfile.js

```diff
@@ -19,11 +19,40 @@ function exec(command, workingDirectory = '.', logCommand = true) {
   }
 }
 
+function checkDirectory(actualList, expectedList) {
+  const extras = actualList.filter(path => !expectedList.includes(path));
+  const missing = expectedList.filter(path => !actualList.includes(path));
+
+  let isValid = true;
+
+  if (extras.length !== 0) {
+    console.error(chalk.white.bgRedBright(`The Assembled directory contains unexpected files:`));
```

Comment:
Maybe pass a directory name arg so we can print out the right directory here (and not just "Assembled")?

Reply:
Fixed.